sct.py

```diff
@@ -185,6 +186,10 @@ def getTestCaseNames(self, testCaseClass):
               help="Install paths for extra python packages to install, scylla-cluster-plugins for example")
 def cli():
     LOGGER.info("install-bash-completion current path: %s", os.getcwd())
+    if not is_using_aws_mock():
+        key_store = KeyStore()
+        key_store.sync(keys=['scylla-qa-ec2', 'scylla-test', 'scylla_test_id_ed25519'],
+                       local_path=Path('~/.ssh/').expanduser(), permissions=0o0600)
```

Comment:
What is this key "scylla_test_id_ed25519"?

Reply:
In rocky9 we started using this key, we should slowly move everything to this type of key, and remove all the tricks we do in ssh servers to enable old types

Reply:
It's an interesting question. I don't think there's a clear consensus of why the newer elliptic-key algorithm is better than the older RSA. Neither are "quantum safe", or offer any other huge advantage over the other. There are some small advantages like key length which doesn't matter to us at all. I think all the hoopla against RSA comes from this - https://www.openssh.com/txt/release-8.2 - it isn't RSA which sucks but SHA-1, and the original openssh RSA implementation used SHA-1 (the current one uses SHA-2 by default, and doesn't suffer from this problem).

Reply:
I care more about things that would work best out of the box, then which key is really superior.

And the rsa ones don't work out of the box, and were causing use issues cause we need to do all kinds of work arounds to enable it